content/blog/2023/09/25/2023-09-25-building-jenkinsio-with-alternative-tools.adoc

```diff
@@ -0,0 +1,83 @@
+---
+:layout: post
+:title: "GSoC Building Jenkins.io with alternative tools"
+:tags:
+- gsoc
+- gsoc2023
+- jenkins.io
+- jenkins-infra
+:author: vandit1604
+:sig: gsoc
+description: >
+  The Building Jenkins.io with Alternative Tools is a project under Google Summer of Code (GSoC) 2023.
+opengraph:
+  image: /images/gsoc/opengraph.png
+links:
+  gitter: https://matrix.to/#/#jenkinsci_gsoc-2023-building-jenkinsio:matrix.org
+---
+
+This blog showcases all the work done in the project link:/projects/gsoc/2023/projects/alternative-jenkinsio-build-tool/[Building Jenkins.io with Alternative Tools] during Google Summer of Code 2023.
+
+== Table of Content
+
+. <<About Project>>
+. Coding <<Phase 1>>
+. Coding <<Phase 2>>
+. <<Future Improvements>>
+. <<Acknowledgments>>
+. <<Useful Links>>
+
+== About Project
+
+The link:/[Jenkins documentation site] is generated as a static website using Awestruct from AsciiDoc sources.
+One of the drawbacks of the current build method is that the technical documentation is not versioned.
+It is thus not possible to view the documentation for a given Jenkins version.
+Only the latest can be viewed.
+This can lead to unnecessary confusion and is a worse experience than many other documentation sites like the link:https://git-scm.com/docs/git[git site], link:https://docs.python.org/3.11/[Python[, and others.
```

Comment:
Correct a mistake that I made in my earlier suggestion
```suggestion
This can lead to unnecessary confusion and is a worse experience than many other documentation sites like the link:https://git-scm.com/docs/git[git site], link:https://docs.python.org/3.11/[Python], and others.
```